functions/Remove-DbaDbView.ps1

```diff
@@ -0,0 +1,118 @@
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: View, Database
+        Author: Mikey Bronowski (@MikeyBronowski), https://bronowski.it
+
+        Website: https://dbatools.io
+        Copyright: (c) 2021 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/Remove-DbaDbView
+
+    .EXAMPLE
+        PS C:\> Remove-DbaDbView -SqlInstance localhost -Database dbname -View "customview1", "customview2"
+
+        Removes views customview1 and customview2 from the database dbname on the local default SQL Server instance.
+
+    .EXAMPLE
+        PS C:\> Remove-DbaDbView -SqlInstance localhost, sql2016 -Database db1, db2 -View view1, view2, view3
+
+        Removes view1, view2, view3 from db1 and db2 on the local and sql2016 SQL Server instances.
+
+    .EXAMPLE
+        PS C:\> Remove-DbaDbView -SqlInstance localhost -Database msdb -IncludeSystemDbs -View view1, view2, view3
+
+        Removes view1, view2, view3 from db1 and db2 from system database (msdb) on the local SQL Server instances.
+
+    .EXAMPLE
+        PS C:\> $servers = Get-Content C:\servers.txt
+        PS C:\> $servers | Remove-DbaDbView -Database db1, db2 -View view1
+
+        Removes view1 from db1 and db2 on the servers in C:\servers.txt file.
+
+    .EXAMPLE
+        PS C:\> $views = Get-DbaDbView -SqlInstance localhost, sql2016 -Database db1, db2 -View view1, view2, view3
+        PS C:\> $views | Remove-DbaDbView
+
+        Removes view1, view2, view3 from db1 and db2 on the local and sql2016 SQL Server instances.
+    #>
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
+    param (
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [object[]]$Database,
+        [object[]]$ExcludeDatabase,
+        [string[]]$View,
+        [parameter(ValueFromPipeline)]
+        [Microsoft.SqlServer.Management.Smo.View[]]$InputObject,
+        [switch]$EnableException
+    )
+
+    process {
+
+        if ((Test-Bound -ParameterName SqlInstance) -and (Test-Bound -Not -ParameterName Database)) {
+            Stop-Function -Message "Database is required when SqlInstance is specified"
+            return
+        }
+
+        if ($SqlInstance) {
+            $InputObject = Get-DbaDbView -SqlInstance $SqlInstance -SqlCredential $SqlCredential -Database $Database -View $View -ExcludeSystemView -ExcludeDatabase $ExcludeDatabase
+        }
+
+        foreach ($vw in $InputObject) {
+
+            if ($Pscmdlet.ShouldProcess($vw.SqlInstance, "Removing the view $vw in the database $($vw.Parent.Name)")) {
```

Comment:
As more than one SqlInstance is allowed, maybe also add the instance (.Parent.Parent.Name) in the message.

Reply:
> As more than one SqlInstance is allowed, maybe also add the instance (.Parent.Parent.Name) in the message.

That's currently my output when I run :
```powershell
Get-dbadbview -SqlInstance $s1,$s2 -ExcludeSystemView -Database Mikey | Remove-DbaDbView -Verbose -Confirm:$false
```

```
VERBOSE: Performing the operation "Removing the view [dbo].[a] in the database Mikey" on target "sql1".
VERBOSE: Performing the operation "Removing the view [dbo].[a] in the database Mikey" on target "sql2".
```

Reply:
Oh, yes - I forgot the target.